reference/constraints/UniqueEntity.rst

```diff
@@ -132,7 +132,11 @@ message
 
 **type**: ``string`` **default**: ``This value is already used.``
 
-The message that's displayed when this constraint fails.
+The message that's displayed when this constraint fails. When using multiple
+fields, the ``{{ vallue }}`` parameter only contains the value of the first
```

Comment:
actually `{{ value }}` represents the entity, but only the first invalid field will trigger a violation to its path